### DISCUSSION TITLE
Suggestions for Managing Blog Posts: Medium Integration vs. Custom Functionality

### DISCUSSION THREAD

**Discussion by fed-tech** [fed-tech/FED-Frontend#252] [Ideas]
Should we integrate Medium posts via its API into our blog post page, or create a custom blog management system on our website? The proposed custom functionality would allow department directors and admins to manage blog posts, similar to how events are handled. Note that the content and media for Medium and our custom system would remain identical.  

We’d love to hear your thoughts and suggestions on which approach would be more effective and why!

Medium api docs refs - https://github.com/Medium/medium-api-docs

**Comment by AnshRaj112** [comment]
Integrating a custom blog management system into website is a more effective approach as it provides complete control over the content, design, and user experience.
